streampark-console/streampark-console-webapp/src/views/system/member/MemberAdd.vue

```diff
@@ -119,10 +117,7 @@ export default {
       this.form.validateFields((err, member) => {
         if (!err && this.validateStatus === 'success') {
           this.loading = true
-          post({
-            ...member,
```

Comment:
The member page cann't be refresh when I switch the team. It may be missed case.

Reply:
> The member page cann't be refresh when I switch the team. It may be missed case.

The teamId will be passed to the backend in all requests .
see  request.js line 80.

Reply:
You can test in your local. The teamId is changed, but front-end doesn't call backend after the teamId is changed.

Reply:
Checked, it's done.